### PR TITLE
Style signup and signin

### PR DESCRIPTION
**Pull request by cmperez**
sorry about the length, it's mostly CSS :sob:

### CODE REVIEW

**Comment by owen-d** [comment]
Oh lord :sob:

**Comment by cmperez** [comment]
looks good tho, I promise

File: client/js/components/signin_form.jsx [hunk 1]
```diff
@@ -33,20 +33,45 @@ var SigninForm = React.createClass({
...
+                  </span>
+                  <input placeholder="Username*" className="input form-username"
+                         type="text" ref="username"/>
+                </div>
+
+                <div className="password-block">
+                  <span className="password-icon">
+                    <i className="fa fa-key"></i>
+                  </span>
+                  <input placeholder="Password*" className="input form-password" id="x"
```
Comment (zindlerb) [inline client/js/components/signin_form.jsx]: id="x" :question:

File: client/js/components/signin_form.jsx [hunk 2]
```diff
@@ -3,13 +3,13 @@ var $ = require("jquery");
 var AppStore = require("../store/app_store.js");
 var router;
```
Comment (zindlerb) [inline client/js/components/signin_form.jsx]: :sob:

**Comment by zindlerb** [comment]
:+1:

File: client/js/components/signin_form.jsx [hunk 3]
```diff
@@ -3,13 +3,13 @@ var $ = require("jquery");
...
 
-var SigninForm = React.createClass({ 
-  
+var SigninForm = React.createClass({
+
   componentWillMount: function(){
```
Comment (dduugg) [inline client/js/components/signin_form.jsx]: whitespace

File: client/js/components/signin_form.jsx [hunk 4]
```diff
@@ -3,13 +3,13 @@ var $ = require("jquery");
...
     this.formDataKeys = ["username", "password"];
     router = this.props.router;
   },
-  
+
   handleSubmit: function(event) {
```
Comment (dduugg) [inline client/js/components/signin_form.jsx]: :clap:

File: client/js/components/signin_form.jsx [hunk 5]
```diff
@@ -33,20 +33,45 @@ var SigninForm = React.createClass({
...
-            Password*: <input type="password"
-                      ref="password"/>
-          <br />
-            <input type="submit" value="Login"/>
-          <br />
-        </form>
+      <div className="table-frame">
+        <div className="table-cell">
+
+          <div className="signin-block">
```
Comment (dduugg) [inline client/js/components/signin_form.jsx]: what does the whitespace padding for some divs but not others indicate?

File: client/js/components/signup_form.jsx [hunk 6]
```diff
@@ -331,70 +331,96 @@ var SignupForm = React.createClass({
         localStorage.setItem("token", data.token);
         router.navigate("/", {trigger: true});
         AppStore.emitChange();
-        },
+      },
```
Comment (dduugg) [inline client/js/components/signup_form.jsx]: :smiley:

File: client/js/components/signup_form.jsx [hunk 7]
```diff
@@ -331,70 +331,96 @@ var SignupForm = React.createClass({
...
-            <input placeholder="Password*" className="form-input-field"
-                   type="password" ref="password"/>
-          </div>
+          <div className="signin-block">
 
-          <div className="form-email">
-            <input placeholder="Email*" className="form-input-field"
-                   type="email" ref="email" />
-          </div>
+            <div className="signup-block-header">
```
Comment (dduugg) [inline client/js/components/signup_form.jsx]: :100:

File: client/js/components/signup_form.jsx [hunk 8]
```diff
@@ -331,70 +331,96 @@ var SignupForm = React.createClass({
...
+                    <i className="fa fa-envelope"></i>
+                  </span>
+                  <input placeholder="Email*" className="input form-email"
+                         type="email" ref="email"/>
+                </div>
+
+                <div className="dropdown signup-form-country">
+                  <p className="signup-form-label">Country*: </p>
+                  <select defaultValue="US" ref="country">
+                          {_.map(this.state.countries, function(countryTuple){
```
Comment (dduugg) [inline client/js/components/signup_form.jsx]: some whitespace issues here

**Pull request merged**